pkg/beholder/pb/keystone/keystone_custom.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/beholder/pb/keystone";
+
+package pb;
+
+message KeystoneCustomMessage {
+  string msg=1;
+  // https://protobuf.dev/programming-guides/proto3/#maps
+  // In go: if Value is empty for a key, nothing will be serialized
+  map<string, Value> labels=2;
```

Comment:
@patrickhuie19 Can we import the values library here instead? We shouldn't need to duplicate this type

Reply:
345595f38e0ec17b8a5f1f488aed12aca0414d86